satpy/tests/reader_tests/test_seviri_l1b_native.py

```diff
@@ -528,7 +528,7 @@ class TestNativeMSGArea(unittest.TestCase):
     """
 
     @staticmethod
-    def create_test_header(earth_model, dataset_id, is_full_disk, is_rapid_scan):
+    def create_test_header(earth_model, dataset_id, is_full_disk, is_rapid_scan, good_qual=True):
```

Comment:
How about having the quality value here directly, eg
```suggestion
    def create_test_header(earth_model, dataset_id, is_full_disk, is_rapid_scan, qual=OK):
```
Then when you call that function, you just pass `qual=NOK` when you want to test the other cases. It would simplify things a little.